enhancements/console/quick-starts.md

```diff
@@ -151,6 +156,62 @@ resource actions. In order for the user to see the Quick Start and
 take its tour, his access review needs to pass all the listed
 resources actions.
 
+#### Quick Start Internationalization
+
+OpenShift can be toggled between multiple languages. The Quick Start
+CRD contains a series of labels to specify the Quick Start name and language:
+```yaml
+console.openshift.io/lang: en
+console.openshift.io/country: gb
+console.openshift.io/name: explore-serverless
+```
+
+The optional `console.openshift.io/lang` label specifies the two-letter
+[ISO 639-2 Language Code](https://www.loc.gov/standards/iso639-2/php/code_list.php))
+(i.e. "en" for English) for the QuickStart. Quick Starts without this label will be treated
+as English Quick Starts.
+
+The optional `console.openshift.io/country` label specifies the alpha-2
+[ISO-3166 Country Code](https://www.iso.org/iso-3166-country-codes.html)
+(i.e. "gb" for the United Kingdom of Great Britain and Northern Ireland).
+Quick Starts without this label will only rely on the `console.openshift.io/lang`
+behavior.
+
+The `console.openshift.io/name: explore-serverless` label specifies the name of the
+Quick Start. This should be consistent across all translations of a given Quick Start,
+as it allows us to avoid showing the same Quick Start content in multiple languages.
+If a Quick Start is not translated into a user's language, the English version will be shown.
+
+OpenShift can display the correct
+Quick Starts for each language by comparing the value of the
+`console.openshift.io/lang` and `console.openshift.io/country` labels to the currently displayed language in OpenShift.
+See the [internationalization enhancement proposal](https://github.com/openshift/enhancements/blob/master/enhancements/console/internationalization.md) for more information.
+This approach will not handle languages we don't offer support for; if Quick Starts
+in non-supported languages become a need, we will need to modify this approach.
```

Comment:
What happens when a QS contains a language in the `console.openshift.io/lang` label that we don't support, eg. `cz` for czech? Is there a fallback? Won't those be used at all?
In this case I would suggest no to use, list, ... those Quick Start CRs. Just use those that are are supported, or don't have the label set, which means that they are english.

Reply:
If we use the existing console approach for i18n, Czech will fall back to English. (We'd be using the same language logic, so the behavior would match.)

We'd get the same ultimate outcome as just using supported languages/Quick Starts that don't have the label set. Happy with either approach.

Reply:
Yeah, I would go with fallback to english in both cases 👍